app/locale/cs.coffee

```diff
@@ -952,11 +952,12 @@ module.exports = nativeDescription: "čeština", englishDescription: "Czech", tr
 #    robin_title: "Senior Product Manager"
 #    nolan_title: "Sales Manager"
 #    lisa_title: "Business Development Manager"
-#    sean_title: "Territory Manager"
 #    liz_title: "Territory Manager"
```

Comment:
Should we delete this line as well as the one beneath with jane?

Reply:
Good catch  - updated